internal/util/cache/local_cache.go

```diff
@@ -186,6 +189,15 @@ func (c *localCache[K, V]) InvalidateAll() {
 
 // Scan entries list with a filter function
 func (c *localCache[K, V]) Scan(filter func(K, V) bool) map[K]V {
```

Comment:
We can't use LRU cache and scan with cache because some of the data might be evicted

Reply:
when are we gonna the list from cache?
can we remove the functionality?

Reply:
there is no guarantee that the scan cache can walk all data, it will be decided by the LRU policy.  the meta cache size is set to Max in rootcoord, so it will not evict.